pandas/core/indexes/datetimelike.py

```diff
@@ -134,7 +135,7 @@ def values(self) -> np.ndarray:
         # Note: PeriodArray overrides this to return an ndarray of objects.
         return self._data._data
 
-    @property
+    @property  # type: ignore
```

Comment:
error: Decorated property not supported
Looks like people are ignoring this error for now. https://github.com/python/mypy/issues/1362